### PR TITLE
Adding the transactions estimated gas limits in constants.py

### PR DESCRIPTION
**Pull request by loredanacirstea**
We can fix https://github.com/raiden-network/raiden-contracts/issues/16 (have a script that generates the gas limit values when compiling the contracts) later.

For now, I just manually added the values to the `constants.py` file, so they can be used in the Raiden client.
Differences from https://github.com/raiden-network/raiden/blob/bc573fc984ce532a85c3ccff96ed733b851f148c/raiden/constants.py are:
- `GAS_REQUIRED_FOR_DISCOVERY_REGISTER = 76000` is now `GAS_REQUIRED_FOR_ENDPOINT_REGISTER = 49183` (not sure where the `76000` value came from)

### CODE REVIEW

File: raiden_contracts/constants.py [hunk 1]
```diff
@@ -25,6 +25,17 @@
...
+GAS_REQUIRED_FOR_OPEN_CHANNEL = 112922
+GAS_REQUIRED_FOR_SET_TOTAL_DEPOSIT = 60110
+GAS_REQUIRED_FOR_CLOSE_CHANNEL = 112715
+GAS_REQUIRED_FOR_UPDATE_BALANCE_PROOF = 96348
+GAS_REQUIRED_FOR_SETTLE_CHANNEL = 125052
+GAS_REQUIRED_FOR_UNLOCK_1_LOCKS = 33515
+GAS_REQUIRED_FOR_UNLOCK_6_LOCKS = 72944
+
+GAS_REQUIRED_FOR_REGISTER_SECRET = 46161
+GAS_REQUIRED_FOR_ENDPOINT_REGISTER = 49183
```
Comment (LefterisJP) [inline raiden_contracts/constants.py]: This will require client changes.

**Comment by hackaugusto** [comment]
These constants most likely change with the version of the smart contract, in the future I believe it makes sense to add them in the json file (which depends on the network and contract versions)

**Comment by LefterisJP** [comment]
@hackaugusto good point.

**Comment by loredanacirstea** [comment]
@hackaugusto , agreed. This is why I kept the original issue open https://github.com/raiden-network/raiden-contracts/issues/16 and explained the state there.

**Pull request merged**